flink/v1.17/flink/src/main/java/org/apache/iceberg/flink/sink/RowDataTaskWriterFactory.java

```diff
@@ -53,6 +55,8 @@ public class RowDataTaskWriterFactory implements TaskWriterFactory<RowData> {
 
   private transient OutputFileFactory outputFileFactory;
 
+  /** @deprecated Use constructor with table supplier instead. */
```

Comment:
should this also say `...will be removed in 1.X.0...` as we do for other deprecated code?